jaxrs/src/main/java/se/fortnox/reactivewizard/ExceptionHandler.java

```diff
@@ -72,7 +73,12 @@ public Publisher<Void> handleException(HttpServerRequest request, HttpServerResp
             webException = new WebException(BAD_REQUEST, "invalidjson", throwable.getMessage());
         } else if (throwable instanceof WebException we) {
             webException = we;
-        } else if (throwable instanceof ClosedChannelException || throwable instanceof AbortedException || throwable instanceof CancellationException) {
+        } else if (throwable instanceof ClosedChannelException
+            || throwable instanceof AbortedException
+            || throwable instanceof CancellationException
+            || (throwable instanceof IOException
```

Comment:
I think we should break out checks that are not just `instanceof` into separate methods.

Reply:
Checks for anticipated exceptions are extracted into a separate method.